cli/nep11_test.go

```diff
@@ -214,7 +213,7 @@ func TestNEP11_OwnerOf_BalanceOf_Transfer(t *testing.T) {
 
 	// tokensOf: good
 	e.Run(t, cmdTokensOf...)
-	e.checkNextLine(t, string(tokenID))
+	require.Equal(t, string(tokenID), e.getNextLine(t))
```

Comment:
Is this really needed?

Reply:
Sometimes there are `+` inside the ID string and `require.Regexp(t, expected, line)` can't handle it. The result looks like this:
```
    executor_test.go:151: 
        	Error Trace:	executor_test.go:151
        	            				executor_test.go:147
        	            				nep11_test.go:217
        	Error:      	Expect "a+L6onljEufAKf15LA7h3YWmhLw=" to match "a+L6onljEufAKf15LA7h3YWmhLw="
```